### ISSUE TITLE
Output scanned format in BarcodeScanner.startScan()

### ISSUE THREAD

**Issue by oneclickwebstudio** [capacitor-community/barcode-scanner#55]
**Is your feature request related to a problem? Please describe.**
What would be the right way to output format after scan result received?

**Describe the solution you'd like**
I have added `found.type` to `PluginCallResultData()` but it prints smth like `"type":"org.gs1.EAN-13"`
```
public func metadataOutput(_ captureOutput: AVCaptureMetadataOutput, didOutput metadataObjects: [AVMetadataObject], from connection: AVCaptureConnection) {

        if (metadataObjects.count == 0 || !self.isScanning) {
            // while nothing is detected, or if scanning is false, do nothing.
            return
        }

        let found = metadataObjects[0] as! AVMetadataMachineReadableCodeObject
        if (targetedFormats.contains(found.type)) {
            var jsObject = PluginCallResultData()
            if (found.stringValue != nil) {
                jsObject["hasContent"] = true
                jsObject["content"] = found.stringValue
                jsObject["type"] = found.type // prints ex:"org.gs1.EAN-13"
            } else {
                jsObject["hasContent"] = false
            }

            if (self.savedCall != nil) {
                savedCall?.resolve(jsObject)
                savedCall = nil
            }

            self.destroy()
        }
    }
```

**Additional context**
How can we convert `fount.type` to format as described in documentation?

<table>
  <thead>
    <tr>
      <th>Category</th>
      <th>Type</th>
      <th>Android</th>
      <th>iOS</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td rowspan="6">1D Product</td>
    </tr>
    <tr>
      <td>UPC_A</td>
      <td>✔</td>
      <td>✔**</td>
    </tr>
    <tr>
      <td>UPC_E</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>UPC_EAN_EXTENSION</td>
      <td>✔</td>
      <td>✖</td>
    </tr>
    <tr>
      <td>EAN_8</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>EAN_13</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td rowspan="8">1D Industrial</td>
    </tr>
    <tr>
      <td>CODE_39</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>CODE_39_MOD_43</td>
      <td>✖</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>CODE_93</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>CODE_128</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>CODABAR</td>
      <td>✔</td>
      <td>✖</td>
    </tr>
    <tr>
      <td>ITF</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>ITF_14</td>
      <td>✖</td>
      <td>✔</td>
    </tr>
    <tr>
      <td rowspan="8">2D</td>
    </tr>
    <tr>
      <td>AZTEC</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>DATA_MATRIX</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>MAXICODE</td>
      <td>✔</td>
      <td>✖</td>
    </tr>
    <tr>
      <td>PDF_417</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>QR_CODE</td>
      <td>✔</td>
      <td>✔</td>
    </tr>
    <tr>
      <td>RSS_14</td>
      <td>✔</td>
      <td>✖</td>
    </tr>
    <tr>
      <td>RSS_EXPANDED</td>
      <td>✔</td>
      <td>✖</td>
    </tr>
  </tbody>
</table>

**Comment by oneclickwebstudio** [comment]
Now i realised it as

```
public func metadataOutput(_ captureOutput: AVCaptureMetadataOutput, didOutput metadataObjects: [AVMetadataObject], from connection: AVCaptureConnection) {

    if (metadataObjects.count == 0 || !self.isScanning) {
        // while nothing is detected, or if scanning is false, do nothing.
        return
    }

    let found = metadataObjects[0] as! AVMetadataMachineReadableCodeObject
    
    if (targetedFormats.contains(found.type)) {
        var jsObject = PluginCallResultData()
        if (found.stringValue != nil) {
            jsObject["hasContent"] = true
            jsObject["content"] = found.stringValue
            jsObject["type"] = formatStringFromMetadata(found.type)
        } else {
            jsObject["hasContent"] = false
        }

        if (self.savedCall != nil) {
            savedCall?.resolve(jsObject)
            savedCall = nil
        }

        self.destroy()
    }
}

@objc func formatStringFromMetadata(_ type: AVMetadataObject.ObjectType) -> String {
    switch type {
    case AVMetadataObject.ObjectType.qr:
        return "QR_CODE";
    case AVMetadataObject.ObjectType.ean8:
        return "EAN_8";
    case AVMetadataObject.ObjectType.ean13:
        return "EAN_13";
    ........

    default:
        return ""
    }
    
}
```

**Comment by jameshhood** [comment]
I was just looking through all the documentation to see how to achieve this and glad I came across this. I have to support a few different types of barcodes and this would be fairly handy for validation. Thanks

**Comment by thegnuu** [comment]
PR #56 is merged and will be released in `2.1.0` next week
